### ISSUE TITLE
Revise print for t-test?

### ISSUE THREAD

**Issue by strengejacke** [easystats/parameters#435]
I have also a more complex output in mind, like the stata output:

![image](https://user-images.githubusercontent.com/26301769/108620627-a5445000-742d-11eb-9511-adc6f37b08c1.png)

_Originally posted by @strengejacke in https://github.com/easystats/parameters/issues/421#issuecomment-782824935_

**Comment by DominiqueMakowski** [comment]
Im wondering, regarding a two+ liners output, because users that expect parameters to "broomify" or "tidy" the summary will definitely expect a one table line, because t-tests have one parameter. Maybe this could be something that we better fit in report, in which we already have functions to summarize samples etc. (And where we are not limited by the output format)?

**Comment by strengejacke** [comment]
Yes, sounds good! I wasn't even thinking about having a larger data frame that is returned. Rather, a more complex print method. But maybe we can indeed move it to report.
